daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/ValueTranslator.scala

```diff
@@ -210,7 +210,7 @@ private[engine] final class ValueTranslator(
                       mbLbl.foreach(lbl_ =>
                         if (lbl_ != lbl)
                           typeError(
-                            s"Mismatching record label $lbl_ (expecting $lbl) for record $tyCon"
+                            s"Mismatching record field label $lbl_ (expecting $lbl) for record $tyCon"
```

Comment:
```suggestion
                            s"Mismatching record field $lbl_ (expecting $lbl) for record $tyCon"
```

Reply:
I disagree with this, this is really about the `name` of the field, i.e., the field label having a mismatch.

Reply:
Ok.